src/screens/regimen_setup/currentProducts/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import Button from '../../../components/button';
+import { useThemeContext } from '../../../theme';
+import applyScale from '../../../utils/applyScale';
+import { Container, Image, Content, Title, SubContent } from './style';
+
+interface Iprops {
+  title: string;
+}
+
+interface stateType {
+  display: 'flex' | 'none';
```

Comment:
Please is this a type or a value? Am confused here bro

Reply:
is a type to use with style{{display}}, style->display property accepts just that

Reply:
but am supposed to delete this one to leave the later